app/controllers/forest_liana/sessions_controller.rb

```diff
@@ -53,7 +55,7 @@ def process_login(
           raise Errors::HTTP401Error
         end
 
-        # TODO: Add ip whitelist retrieving when it exists.
+        IpWhitelist.retrieve
```

Comment:
I would add a comment to explain why we do this specific `IpWhitelist.retrieve` here.